src/gateway.c

```diff
@@ -513,10 +519,11 @@ static int handle_exec(struct gateway *g, struct handle *req)
 	}
 	req->stmt_id = stmt->id;
 	g->req = req;
-	req_id = idNext(&g->random_state);
```

Comment:
Why are we removing the concept of the id? I can still see several instances of `req->id` for example in leader.c:281 and we are not removing the field from the struct, why?

Reply:
`idNext` et al are code that I introduced a while ago to provide a way of tracking requests in traces. Now we have `sm.c` that does the same thing better (generalizes to all kinds of requests, `sm_relate`, no RNG), so I thought it was time to get rid of it. I'll clean up the remaining mentions of the field, thanks for spotting

Reply:
Done